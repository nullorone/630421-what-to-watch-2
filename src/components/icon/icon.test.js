import React from "react";
import renderer from "react-test-renderer";
import Icon from "./icon";

describe(`Make snapshot`, () => {
  const initProps = {
    name: `PAUSE`,
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <Icon {...initProps}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
