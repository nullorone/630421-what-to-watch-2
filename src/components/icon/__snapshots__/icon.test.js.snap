// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<symbol
  id="pause"
  viewBox="0 0 14 21"
>
  <g
    fill="none"
    fillRule="evenodd"
    id="Artboard"
    stroke="none"
    strokeWidth="1"
  >
    <polygon
      fill="#EEE5B5"
      fillRule="nonzero"
      id="Line"
      points="0 -1.11910481e-13 4 -1.11910481e-13 4 21 0 21"
    />
    <polygon
      fill="#EEE5B5"
      fillRule="nonzero"
      id="Line"
      points="10 -1.11910481e-13 14 -1.11910481e-13 14 21 10 21"
    />
  </g>
</symbol>
`;
